src/Restaurant/Domain/Restaurant.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Restaurant\Domain;
+
+use App\Core\Domain\AggregateRoot;
+use App\Restaurant\Domain\Collection\Menu;
+use App\Restaurant\Domain\ValueObject\Dish\Description;
+use App\Restaurant\Domain\ValueObject\Dish\Picture;
+use App\Restaurant\Domain\ValueObject\Dish\Price;
+use App\Restaurant\Domain\ValueObject\Dish\Weight;
+use App\Restaurant\Domain\ValueObject\Dish\Name as DishName;
+use App\Restaurant\Domain\ValueObject\Restaurant\Delivery;
+use App\Restaurant\Domain\ValueObject\Restaurant\Name;
+use App\Restaurant\Domain\ValueObject\Restaurant\RestaurantId;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table(name="restaurants")
+ */
+class Restaurant extends AggregateRoot
```

Comment:
In our case this class has strange methods as add dish remove dish. In the simple case we replace menu every day, but in your case we have menu and every day we edit dishes in the menu. Try to think in the real process cases